walkthroughs/howto-k8s-alb/README.md

```diff
@@ -59,7 +59,7 @@ You should see similar to the following.
       Normal  CREATE  11m    alb-ingress-controller  LoadBalancer 80113f18-howtok8salb-color-0f20 created, ARN: arn:aws:elasticloadbalancing:us-west-2:669977933099:loadbalancer/app/80113f18-howtok8salb-color-0f20/7c45f6fd4eefa871
```

Comment:
we might need to the alb-ingress-controller, confirm it.

Reply:
updated